src/components/NcHeaderMenu/NcHeaderMenu.vue

```diff
@@ -413,8 +413,8 @@ $externalMargin: 8px;
 	.header-menu {
 		width: $clickable-area;
 
-		&__trigger {
-			width: $clickable-area;
+		#{&}__trigger {
+			width: 100% !important;
```

Comment:
Seems that this is 100% on any screen, so we can remove this from `@media` and set it above instead of `width: var(--header-height) !important;`

(haven't tested)

Reply:
Motivation: to have less source or truth

Reply:
Tested this out on my local dev - it works! Sorry for getting back so late, I discarded this notification by accident 😅

![firefox_thFKoqAqEa](https://github.com/nextcloud-libraries/nextcloud-vue/assets/110193237/e0652f0f-20d0-4997-b712-e90ee7510266)